sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatClient.java

```diff
@@ -7,8 +7,8 @@
 import com.azure.android.communication.chat.models.CreateChatThreadOptions;
 import com.azure.android.communication.chat.models.CreateChatThreadResult;
 import com.azure.android.communication.chat.models.ListChatThreadsOptions;
-import com.azure.android.communication.chat.signaling.RealTimeNotificationCallback;
-import com.azure.android.communication.chat.signaling.properties.ChatEventId;
+import com.azure.android.communication.chat.models.signaling.RealTimeNotificationCallback;
+import com.azure.android.communication.chat.models.signaling.ChatEventId;
```

Comment:
Should we flatten these models instead of nesting? i.e. completely removing `signaling` from public surface

Reply:
That's an option as well. Would adding a signaling package make the code more readable?

Reply:
We decided to merge notification APIs with ChatClient, so I was thinking, why we want to give user an impression that notification is something special even after the merge; For instance, user write the below code:

```java
communication.chat.ChatAsyncClient chatAsyncClient = ..

communication.chat.model.CreateChatThreadResult result = chatAsyncClient.createChatThread()..;
```

```java
chatAsyncClient.on(threadId, new communication.chat.model.signaling.RealTimeNotificationCallback() {
   ...
   communication.chat.model.signaling.BaseEvent = ..
});
```

from user side, he is  using a single client but some of the methods in the same client uses models from one namespace, some from nested model namespace, just wondering what value this separation brings.